tests/all/utils/fixture.rs

```diff
@@ -87,14 +144,29 @@ impl Fixture {
                 }).unwrap();
         });
 
-        copy_dir(bin, self.path.join("bin")).expect("could not copy `bin` directory into temp dir");
+        assert!(shared_wasm_bindgen.is_file());
+        assert!(shared_wasm_bindgen_test_runner.is_file());
+
+        wasm_pack::binaries::ensure_local_bin_dir(&self.path).unwrap();
+
+        fs::copy(
+            &shared_wasm_bindgen,
+            wasm_pack::binaries::local_bin_path(&self.path, "wasm-bindgen"),
+        ).expect("could not copy `wasm-bindgen` to fixture directory");
+
+        fs::copy(
```

Comment:
For an even speedier test I'd recommend using `hard_link` first and falling back to `copy` only if that fails

Reply:
done!